data/soapvalidator/Mail/Bugs/ZBUG-3364.xml

```diff
@@ -0,0 +1,516 @@
+<t:tests xmlns:t="urn:zimbraTestHarness">
+
+<t:property name="test_account1.name" value="test1.${TIME}.${COUNTER}@${defaultdomain.name}"/>
+<t:property name="test_account2.name" value="test2.${TIME}.${COUNTER}@${defaultdomain.name}"/>
```

Comment:
is this account used for delegated testcases?

Reply:
test_account1.name is the account which is part of DL and has sendAsDistList right. I will add few more test cases in the next sprint so keep test_account2.name as well ZCS-13336. If needed I can delete it - test_account2.name.